instrumentation/opentelemetry-annotations-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/otelannotations/async/CompletableFutureMethodSpanStrategy.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.otelannotations.async;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.tracer.BaseTracer;
+import java.util.concurrent.CompletableFuture;
+
+enum CompletableFutureMethodSpanStrategy implements MethodSpanStrategy {
```

Comment:
Isn't `CompletableFuture` just an implementation of `CompletionStage`? Wouldn't `CompletionStageMethodSpanStrategy` work for both of them?